ui/client/assets/json/nodeAttributes.json

```diff
@@ -35,7 +35,7 @@
     }
   },
   "Switch": {
```

Comment:
What about this?

Reply:
Switch component type (in scenario json) stay with us. We change only displayed labels on FE.

Reply:
Ehh this will be misleading..